portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +28,33 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+var slideIndex = 1;
+showSlides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("thumbimgcl");
+  var captionText = document.getElementById("caption");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
```

Comment:
When doing single line if statements, the braces should be omitted. But what I would rather see, is you keep the braces and expand it to be multiple lines. Following the bracing style [here](https://google.github.io/styleguide/jsguide.html#formatting-braces). Internally we generally don't do single line block statements even if it is really simple.

no need to fix this here, but try to do it in the future.